### ISSUE TITLE
3 digit modes are incorrectly flagged

### ISSUE THREAD

**Issue by erikPrime** [rodjek/puppet-lint#431]
In the below example, an example site.pp is incorrectly flagged as needing a 4 digit octal value.  Three digit octal values are valid in the case where the user does not wish to change the sticky bit.

```
filebucket { 'main':
  server => 'fwhqw4xkniw9t3f.delivery.puppetlabs.net',
  path   => false,
}

File { backup => 'main' }

node default {

file { '/tmp/exec_path_multi_string':
  ensure => directory,
  mode   => '777'
}

}

[root@fwhqw4xkniw9t3f manifests]# puppet-lint site.pp
WARNING: mode should be represented as a 4 digit octal value or symbolic mode on line 12>

**Comment by ghoneycutt** [comment]
@erikPrime That fourth digit does more than a sticky bit, anyhow the style guide states that one must use four digit modes. This is because we want to be exact in the description of the resource. In your case, use `0777`.

http://docs.puppetlabs.com/guides/style_guide.html#file-modes
